app/app.rb

```diff
@@ -0,0 +1,79 @@
+ENV["RACK_ENV"] ||= 'development'
+
+require 'sinatra/base'
+require 'sinatra/flash'
+require_relative 'data_mapper_setup'
+
+class Chitter < Sinatra::Base
+  use Rack::MethodOverride
+  enable :sessions
+  set :session_secret, 'shhh'
+  register Sinatra::Flash
+
+  get '/peeps' do
+    @peeps = Peep.all
+    erb :'peeps/index'
+  end
+
+  get '/peeps/new' do
+    erb :'peeps/new'
+  end
+
+  post '/peeps' do
+    peep = Peep.create(message: params[:message])
```

Comment:
Are lines 23 to 26 controller code? (Are they concerned with messaging between the view and model?)